seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-ftp-e2e/src/test/resources/excel/fake_source_to_ftp_excel.conf

```diff
@@ -81,5 +81,6 @@ sink {
     file_format_type = "excel"
     filename_time_format = "yyyy.MM.dd"
     is_enable_transaction = true
+    connection_mode=active_local
```

Comment:
since the default value is `active_local`. So I think we can remove it from config.

Reply:
Modified , thanks